### ISSUE TITLE
Feature: Allow routing demonstration

### ISSUE THREAD

**Issue by bebraw** [christianalfoni/webpack-bin#133]
I'm not sure if it's technically possible, but it would be neat if it webpackbin allowed routing demonstrations. I'm thinking mainly react-router. I don't know what this would take technically, though.

**Comment by christianalfoni** [comment]
Yeah, this is possible I believe, just have to expose a fake addressbar for the iframe and back/forward buttons. Have to research a bit
